src/helpers/files.js

```diff
@@ -24,13 +24,15 @@ function manualBlacklist() {
     '.nvmrc',
     '.gitignore',
     '.DS_Store',
+    'vendor',
   ]
 }
 
 function globBlacklist() {
   // TODO: honor the .gitignore file instead of a hard-coded list
   return [
     'node_modules/**/*',
```

Comment:
@drazisil-codecov , why do you do the one directory this way?

Reply:
lolol

Reply:
I was just copying what was there for other dirs. Not sure the best approach here.

Reply:
I'm the last person to ask about globs. In theory my tests says both ways work 😇 🤷‍♀️